src/index.js

```diff
@@ -20,19 +20,62 @@ inputList.addEventListener('keydown', (event) => {
   }
 });
 
+// Delete button
+function deleteItem(i) {
+  taskArray.splice(i, 1);
+  localStorage.setItem('tasks', JSON.stringify(taskArray));
+  window.location.reload();
+}
+function actRemove() {
+  const removeBtn = document.querySelectorAll('.remove-Btn');
+  removeBtn.forEach((db, i) => {
+    db.addEventListener('click', () => { deleteItem(i); });
+  });
+}
+
+function checkBox() {
+  const checks = document.querySelectorAll('input[type=checkbox]');
+  const inputs = document.querySelectorAll('.strike');
+  checks.forEach((ck, i) => {
+    ck.addEventListener('change', () => {
+      if (checks[i].checked) {
+        inputs[i].style.textDecoration = 'line-through';
+        inputs[i].style.color = 'grey';
+      } else {
+        inputs[i].style.textDecoration = 'none';
+        inputs[i].style.color = 'black';
+      }
+    });
+  });
+}
+
 function displayItems() {
   let tasks = '';
   for (let i = 0; i < taskArray.length; i += 1) {
     tasks += ` <div class="item" >
-    <div class="input-Content">
-    <div>
-        <input type="checkbox" class="complete" name="completed" />
-        <span disabled>${taskArray[i]}</span>
+    <div class = 'task-Container'>
+        <input type="checkbox" class="complete stylingInput" name="completed" />
+        <input class="stylingInput strike" type="text" placeholder="${taskArray[i]}"/>
+        <div class='edit-Btn'>
+          <div class='editBtn1'></div>
+          <div class='editBtn1'></div>
+          <div class='editBtn1'></div>
+        </div>
     </div>
```

Comment:
- [ ] You have done great to implement the addition and display of the task 👏 . However, I have noticed that the tasks added are not added to the local storage and therefore this affects the task display 😞 . Therefore I suggest that you ensure that the tasks are added and displayed properly. In addition, ensure that the edition and deletion functionalities are also working properly 🥰 .

Video demo 😎